code/modules/tgs/v3210/api.dm

```diff
@@ -0,0 +1,244 @@
+#define REBOOT_MODE_NORMAL 0
```

Comment:
I don't give a shit about ancient tgs either

Reply:
It's part of the package, will eventually be removed when MSO stops being a nerd about it.

Reply:
I mean it's either you remove it or I commit to this branch to do so

Reply:
Does it really matter? These files are `#ifdef` excluded already (https://github.com/goonstation/goonstation/pull/18297/files#diff-fdad9a583d15fb729dd330a8f5e44935fd9e28c4570d0628fdad1e99635edd01R6). Modifying the code just creates an unnecessary diff between the official release.

Reply:
Guess this PR is sitting around till @MrStonedOne updates off V3 cause can't strip it from source until then.